paper/src/main/java/com/denizenscript/denizen/paper/events/UnknownCommandScriptEvent.java

```diff
@@ -53,60 +54,37 @@ public class UnknownCommandScriptEvent extends BukkitScriptEvent implements List
 
     public UnknownCommandScriptEvent() {
         registerCouldMatcher("command unknown");
+        this.<UnknownCommandScriptEvent, ElementTag>registerDetermination(null, ElementTag.class, (evt, context, text) -> {
+            evt.event.message(PaperModule.parseFormattedText(text.toString(), ChatColor.WHITE));
+        });
+        this.<UnknownCommandScriptEvent>registerTextDetermination("none", (evt) -> {
+            evt.event.message(null);
+        });
     }
 
     public UnknownCommandEvent event;
     public String command;
     public String rawArgs;
     public String sourceType;
 
-    @Override
-    public boolean applyDetermination(ScriptPath path, ObjectTag determinationObj) {
-        if (determinationObj instanceof ElementTag) {
-            String determination = determinationObj.toString();
-            if (CoreUtilities.equalsIgnoreCase(determination, "none")) {
-                event.setMessage(null);
-            }
-            else {
-                event.setMessage(determination);
-            }
-            return true;
-        }
-        return super.applyDetermination(path, determinationObj);
-    }
-
     @Override
     public ScriptEntryData getScriptEntryData() {
-        return new BukkitScriptEntryData(event.getSender() instanceof Player ? new PlayerTag((Player) event.getSender()) : null, null);
+        return new BukkitScriptEntryData(event.getSender() instanceof Player player ? new PlayerTag(player) : null, null);
     }
 
     @Override
     public ObjectTag getContext(String name) {
-        if (name.equals("command")) {
-            return new ElementTag(command);
-        }
-        else if (name.equals("raw_args")) {
-            return new ElementTag(rawArgs);
-        }
-        else if (name.equals("args")) {
-            return new ListTag(Arrays.asList(ArgumentHelper.buildArgs(rawArgs, false)));
-        }
-        else if (name.equals("server")) {
-            return new ElementTag(sourceType.equals("server"));
-        }
-        else if (name.equals("source_type")) {
-            return new ElementTag(sourceType);
-        }
-        else if (name.equals("command_block_location") && sourceType.equals("command_block")) {
-            return new LocationTag(((BlockCommandSender) event.getSender()).getBlock().getLocation());
-        }
-        else if (name.equals("command_minecart") && sourceType.equals("command_minecart")) {
-            return new EntityTag((CommandMinecart) event.getSender());
-        }
-        else if (name.equals("message")) {
-            return new ElementTag(event.getMessage());
-        }
-        return super.getContext(name);
+        return switch (name) {
+            case "command" -> new ElementTag(command, true);
+            case "raw_args" -> new ElementTag(rawArgs, true);
+            case "args" -> new ListTag(Arrays.asList(ArgumentHelper.buildArgs(rawArgs, false)));

```

Comment:
This has the same issue as `raw_args` - need to pass `, true` to the `ListTag` constructor so that the elements within it are marked as plain-text